app/components/appcomponents/Footer.js

```
@@ -216,35 +216,37 @@ export default function Footer() {
                       src={"/emailco.svg"}
                       alt="Facebook Icon"
                       width={233}
                       height={72}
-                    />
+                    /> */}
+                    {/* </div> */}
+                    <a
+                      href="https://www.facebook.com/osmrtnicacom/"
+                      className="hidden tablet:flex desktop:flex items-center text-gray-600 mobile:mt-[15px] tablet:mt-[10px] desktop:mt-[10px] hover:text-blue-500 transition duration-200 tablet:mt-[45px]"
+                      target="_blank"
+                    >
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)